slices/rabbitmq-server.yaml

```diff
@@ -0,0 +1,62 @@
+package: rabbitmq-server
+
+essential:
+  - rabbitmq-server_copyright
+
+slices:
+  bins:
+    essential:
+      - adduser_bins
+      - base-files_base
+      ## erlang* and logrotate packages do not exist for i386
+      ## for all other archs, uncomment all those slices below
+      # - erlang-base_bins
+      # - erlang-crypto_libs
+      # - erlang-eldap_libs
+      # - erlang-inets_bins
+      # - erlang-mnesia_libs
+      # - erlang-os-mon_libs
+      # - erlang-parsetools_libs
+      # - erlang-public-key_libs
+      # - erlang-runtime-tools_libs
+      # - erlang-ssl_libs
+      # - erlang-syntax-tools_libs
+      # - erlang-tools_libs
+      # - erlang-xmerl_libs
+      # - logrotate_bins
+      - openssl_bins
+      - python3.12_standard
+      - rabbitmq-server_config
+      - rabbitmq-server_libs
+      - socat_bins
+    contents:
+      /usr/bin/rabbitmqadmin:
+      /usr/lib/rabbitmq/bin/rabbitmq-*:
+      /usr/lib/rabbitmq/bin/vmware-rabbitmq:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/sbin/rabbitmq-*:
+      /usr/lib/rabbitmq/lib/rabbitmq_server-3.12.1/sbin/vmware-rabbitmq:
+      /usr/sbin/rabbitmq-*:
+
+  config:
+    contents:
+      /etc/default/rabbitmq-server:
+      /etc/init.d/rabbitmq-server:
+      /etc/logrotate.d/rabbitmq-server:
+      /etc/rabbitmq/rabbitmq-env.conf: {symlink: /usr/share/rabbitmq-env.conf}
+      /usr/lib/rabbitmq/autocomplete/*:
```

Comment:
Seems to be autocomplete stuff for bash and zsh. In many slices in the past, we have not provided these. But if you do want these to be present, let's please include them in another slice e.g. `autocomplete`.

Reply:
moved to a separate slice